google-cloud-firestore/src/main/java/com/google/cloud/firestore/BasePath.java

```diff
@@ -93,6 +94,24 @@ boolean isPrefixOf(B path) {
     return true;
   }
 
+  /**
+   * Compare the current path against another Path object.
```

Comment:
Maybe also note that this is lexographic ordering?

Reply:
Done